Source/Scene/Scene.js

```diff
@@ -1491,7 +1491,7 @@ define([
 
         if (scene.debugShowCommands || scene.debugShowFrustums) {
             executeDebugCommand(command, scene, passState);
-        } else if (scene.frameState.shadowHints.shadowsEnabled && command.receiveShadows) {
+        } else if (scene.frameState.shadowHints.shadowsEnabled && command.receiveShadows && defined(command.derivedCommands.shadows)) {
```

Comment:
Can you add a comment here like the GiHub comment since the overall order of operations may be non-obvious to those new to the code?